webapp/sso.py

```diff
@@ -51,7 +51,9 @@ def logout():
     def before_request():
         if flask.request.path in ["/login", "/logout"]:
             return
-        if flask.request.path.startswith("/guides"):
+        if flask.request.path.startswith(
+            "/guides"
```

Comment:
@anthonydillon are we cool with adding this pack behind SSO for now? Or should we hold off entirely until we've decided what to make public?

Reply:
Yes behind SSO for now but we should be book time in soon to unblock this content